docs/input/index.md

```diff
@@ -0,0 +1,15 @@
+<script setup>
+  import InputTabs from './InputTabs.vue'
+</script>
+
+# input
```

Comment:
It's just my opinion - since we think React components will be used the most by Warp's early adapters, maybe we should consider naming this section based on Fabric React's docs, i.e. instead of `input`, we'd display `TextField` or at least `Field`. Wdyt?

Reply:
Fixed here https://github.com/warp-ds/tech-docs/pull/10/commits/b505a31bf7853cc00189c8896cb030a8d99ba74a